lib/bolt/bolt_option_parser.rb

```diff
@@ -338,10 +338,12 @@ def self.examples(cmd, desc)
       Available options are:
     PROJECT_INIT_HELP
 
+    attr_reader :warnings
     def initialize(options)
       super()
 
       @options = options
+      @warnings = {}
```

Comment:
I made this a hash where key is the option and value is the message thinking we might want to know what option is associated with what message in the cli. But i dont actually use the keys currently, should it just be an array?

Reply:
It seems like for now we typically include the option in the message itself, so an array seems sufficient. Do you have a use case in mind for passing the associated option?

Reply:
Perhaps an array of hashes?

```rb
@warnings = []
# ...
@warnings << { option: 'sudo-password', msg: 'Optional parameter for --sudo-password is deprecated and will no longer prompt for password.' }
```